tests/test_ui_stack/src/main.rs

```diff
@@ -24,12 +24,17 @@ fn main() -> eframe::Result {
 }
 
 #[derive(Default)]
-struct MyApp {}
+struct MyApp {
+    settings: bool,
+    inspection: bool,
+    memory: bool,
```

Comment:
nit: those could be slightly more explicit
```suggestion
    show_settings: bool,
    show_inspection: bool,
    show_memory: bool,
```